tests/test_loadReferenceObjects.py

```diff
@@ -26,24 +26,24 @@
 
 import lsst.afw.table as afwTable
 import lsst.log
-from lsst.meas.algorithms import ReferenceObjectLoaderBase, getRefFluxField, getRefFluxKeys
+from lsst.meas.algorithms import ReferenceObjectLoader, getRefFluxField, getRefFluxKeys
 from lsst.meas.algorithms.loadReferenceObjects import hasNanojanskyFluxUnits, convertToNanojansky
 import lsst.pex.config
 import lsst.utils.tests
 
 from ingestIndexTestBase import makeConvertConfig
 
 
-class TrivialLoader(ReferenceObjectLoaderBase):
-    """Minimal subclass of ReferenceObjectLoaderBase to allow instantiation
+class TrivialLoader(ReferenceObjectLoader):
+    """Minimal subclass of ReferenceObjectLoader to allow instantiation
```

Comment:
You can just delete `TrivialLoader`, since ReferenceObjectLoader is no longer an ABC.